chain/actors/builtin/verifreg/actor.go.template

```diff
@@ -20,14 +20,58 @@ import (
 	"github.com/filecoin-project/lotus/chain/actors/adt"
 	"github.com/filecoin-project/lotus/chain/actors"
 	"github.com/filecoin-project/lotus/chain/types"
-    verifregtypes "github.com/filecoin-project/go-state-types/builtin/v9/verifreg"
+	verifregtypes "github.com/filecoin-project/go-state-types/builtin/v9/verifreg"
+	verifregtypes13 "github.com/filecoin-project/go-state-types/builtin/v14/verifreg"
+)
+
+const (
+	NoAllocationID                      = verifregtypes.NoAllocationID
+	MaximumVerifiedAllocationExpiration = verifregtypes.MaximumVerifiedAllocationExpiration
+	MinimumVerifiedAllocationTerm       = verifregtypes.MinimumVerifiedAllocationTerm
+	MaximumVerifiedAllocationTerm       = verifregtypes.MaximumVerifiedAllocationTerm
+	MinimumVerifiedAllocationSize       = verifregtypes13.MinimumVerifiedAllocationSize
 )
 
 var (
 	Address = builtin{{.latestVersion}}.VerifiedRegistryActorAddr
 	Methods = builtin{{.latestVersion}}.MethodsVerifiedRegistry
 )
 
+type (
+	RemoveDataCapProposal          = verifregtypes.RemoveDataCapProposal
```

Comment:
removed the ones I don't currently need, I'd prefer comprehensiveness so the temptation isn't to reach in to go-state-types directly when one's needed, and instead of doing them all I just started here